controllers/space/space_controller_test.go

```diff
@@ -57,6 +57,7 @@ func TestCreateSpace(t *testing.T) {
 			Exists().
 			HasStatusTargetCluster("member-1").
 			HasConditions(spacetest.Provisioning()).
+			HasStateLabel("cluster-assigned").
```

Comment:
can we use a constant for `cluster-assigned`?

Reply:
I didn't use the constant intentionally - so we know when the content has changed (for example by mistake)